### ISSUE TITLE
When Oxygen is deactivated then add back simple HTML version of content

### ISSUE THREAD

**Issue by dmccan** [soflyy/oxygen-bugs-and-features#2688]
**Describe the feature you'd like to see included in Oxygen.**
Tested with version 4.0 alpha 2.

Currently, if Oxygen is used for post, page, or CPT content (Inner Content area) and the Oxygen Builder plugin is disabled / deactivated then the content is lost.  When Oxygen is deactivated then please add the content back in simple format.  

**What are the use cases for this feature?**
If you create a template and then disable the builder there is no expectation that the template is saved.  However, if you create content and change builders then the content should not be lost.  

I believe that most people understand that you shouldn't use a page builder for your post content, but it is very common to use one for creating pages.  It doesn't make sense to install another page builder on top of Oxygen since Oxygen is a great builder.  

**Examples of this feature or functionality.**
A number of modern builders do this now.  Beaver Builder is a good example.  You get you content as simple unstyled HTML.  This way no content is lost.

**Comment by Spellhammer** [comment]
> Currently, if Oxygen is used for post, page, or CPT content (Inner Content area) and the Oxygen Builder plugin is disabled / deactivated then the content is lost.

This is not true. The content is still available in the ct_builder_json post meta field. This field is not deleted when Oxygen is removed. And since it's JSON, it's very easy to load it into an IDE or JSON viewer and navigate the tree to extract content.

> When Oxygen is deactivated then please add the content back in simple format.

I can't say whether we'll do this, but if we do, it'll be much easier now that we're using JSON instead of shortcodes.

**Comment by dmccan** [comment]
Thank  you for the clarification.  That is an important distinction.  It should read:

Currently, if Oxygen is used for post, page, or CPT content (Inner Content area) and the Oxygen Builder plugin is disabled / deactivated then the content ~~is lost~~ no longer shows, though it is still in the database in a post meta field as JSON. When Oxygen is deactivated then please add the content back in simple format.
